### PR TITLE
Add Conviction graph on Level-5

### PR DESCRIPTION
**Pull request by santteegt**
This PR adds the conviction graph into Level-5 of the game based on requirements:

- [Wireframes](https://www.figma.com/proto/cuFTOOajO4aouz4SLnIINz/Commons-Simulator?node-id=804%3A143&scaling=min-zoom)
- [Mathematical formula](https://www.desmos.com/calculator/w2jgqtqmwjFormula)

### CODE REVIEW

**Comment by mendesfabio** [comment]
Pretty cool, @santteegt! You could add xPadding, yPadding, cornerRadius, and backgroundColor props to the tooltip, so it'd look more like our wireframe. Something like this,

```
 xPadding: 24,
yPadding: 12,
cornerRadius: 0,
backgroundColor: 'rgb(44,44,44)', 
```
What do you think about it?

**Comment by mendesfabio** [comment]
Oh, that's a minor detail, but you could change `halflife` to `halfLife`just to follow the camelCase style.

File: src/views/level5/Level5-1.vue [hunk 1]
```diff
@@ -113,35 +113,103 @@ export default {
...
     chart() {
-      const input = this.forms.input.decisions
+      const input = this.forms.input.decisions && this.forms.input.decisions > 2 ? this.forms.input.decisions : 3
+      const H = this.$store.state.CommonsModule.conviction.halflife
+      const H2 = this.$store.state.CommonsModule.conviction.inflectionPoint
+      const alpha = (1 - H) ** (1 / input)
+      const S = 1 - alpha
+      const t = Math.log((((alpha - 1) * H2) + S) / S) / Math.log(alpha)
+      const daysIndex = [3, 9, 15, 30, 60, 61].filter(x => x <= input).map((x, i) => i)
+      const maxBound = [45, 60, 180, 360, 360, 720][daysIndex.pop()]
```
Comment (sembrestels) [inline src/views/level5/Level5-1.vue:123]: I would set a fixed x axis of 180 days to make sure everyone understand the plot at the first glance.

It's just changing these two lines by `const maxBound = 180`.

**Review by sembrestels — COMMENTED** [review]
You did a fantastic job @santteegt, I think it's mostly done and almost ready to merge. Please take a look to @mendesfabio's suggestion and the one i do below.

**Comment by santteegt** [comment]
Hi @mendesfabio @sembrestels,

Thanks for your feedback. I just updated my PR

**Comment by sembrestels** [comment]
LGTM! What do you think @mendesfabio?

**Comment by mendesfabio** [comment]
LGTM! Congrats again @santteegt.

**Comment by sembrestels** [comment]
Thanks for the feedback @mendesfabio! And I agree, congrats @santteegt!

**Pull request merged**